File: BilanganTerbesar.py
print("Menentukan nilai terbesar 3 bilangan")
print("===================================== \n")

bil1 = int(input("Masukkan bilangan pertama : "))
bil2 = int(input("Masukkan bilangan kedua : "))
bil3 = int(input("Masukkan bilangan ketiga : "))

terbesar = int
terbesar = bil1

if (bil2>terbesar):
    terbesar = bil2
    
if (bil3>terbesar):
    terbesar = bil3
    
print("\nBilangan terbesar adalah " + str(terbesar))

if (bil1==bil2):
    print("Bilangan pertama sama dengan bilangan kedua")
if (bil2==bil3):
    print("Bilangan kedua sama dengan bilangan ketiga")
if (bil1==bil3):
    print("Bilangan pertama sama dengan bilangan ketiga")

File: BilanganTerbesar.java
import java.util.Scanner;
class Main {
   public static void main(String[] args) {
       System.out.println("Menentukan nilai terbesar 3 bilangan");
       System.out.println("=====================================");
       
        Scanner input=new Scanner(System.in);
        System.out.println("\nMasukkan bilangan pertama : ");
        int Bil1=input.nextInt();
        System.out.println("Masukkan bilangan kedua : ");
        int Bil2=input.nextInt();
        System.out.println("Masukkan bilangan ketiga : ");
        int Bil3=input.nextInt();
        
        int terbesar;
        terbesar = Bil1;
        
        if(Bil2>terbesar){
            terbesar = Bil2;
        }
        if(Bil3>terbesar){
            terbesar = Bil3;
        }
            System.out.println("\nBilangan terbesar adalah " + terbesar);
        
        
        if (Bil1==Bil2){
            System.out.println("Bilangan pertama sama dengan bilangan kedua");
        }
        if (Bil2==Bil3){
            System.out.println("Bilangan kedua sama dengan bilangan ketiga");
        }
        if (Bil1==Bil3){
            System.out.println("Bilangan pertama sama dengan bilangan ketiga");
        }
        
    }
}    


